sdk/textanalytics/azure-ai-textanalytics/samples/README.md

```diff
@@ -54,8 +54,8 @@ For more information about how the versioning story of the SDK corresponds to th
   pip install azure-identity
   ```
 
-2. Clone or download this sample repository
-3. Open the sample folder in Visual Studio Code or your IDE of choice.
+2. Clone or download this sample file
+3. Open the sample file in Visual Studio Code or your IDE of choice.
```

Comment:
Hi @BaherAbdullah, thanks for your PR. Would you mind explaining the need for the updated language? It is intended to explain that you can clone the repo and open the samples folder to run any sample, but let me know if you think that it is confusing.

Reply:
Hi @kristapratico , I felt its a bit confusing using the word repository, because I felt I had to download a full repo and not a single file. Following this, I felt you should open the file and not a folder.

Reply:
Thanks for the feedback. I can see how it can be kind of confusing to have to clone the whole repo just to access the sample files. Would you mind tweaking the sentence slightly for clarity to something like this? Thank you so much!

```suggestion
2. Clone the repo or download the sample file
3. Open the sample file in Visual Studio Code or your IDE of choice.
```
```suggestion
2. Clone or download this sample file
3. Open the sample file in Visual Studio Code or your IDE of choice.
```